journal-short.bib

```diff
@@ -75,6 +76,7 @@ @string{ RAS
 @string{ SCHOLAR     = "Scholarpedia" }
 @string{ SCI         = "Science" }
 @string{ SCIROB      = "Sci. Robot." }
+@string{ SWEVO       = "Swarm and Evolutionary Computation" }
 @string{ SI          = "Swarm Intell." }
 @string{ SNCS        = "SN Comput. Sci." }
 @string{ SWEVO       = "Swarm and Evolutionary Computation" }
```

Comment:
SWEVO is already part of this file. Also, the ordering of this entry does not make sense to me. Either we sort by key, then SWEVO should come after SI and SNCS, or we sort by name of the journal, then SN should come before Swarm?

Reply:
Indeed, the names of the journals are still quite messy. I propose that, since the name of the keys (to some extent) random, we keep the most consistent information as ordering element: the name of the journal.

We will need to rework a bit the two journal bib files. What do you think?

Reply:
Hi, I reordered the entries in journal.bib and journal-short.bib, and corrected two duplicated entries.

I propose that the the order of journal.bib takes precedence over the order that could arise with the abbreviations in journal-short.bib.

I think we can revise the abbreviations for the entries of journal-short.bib in a different pull request.